main.cpp

```diff
@@ -0,0 +1,130 @@
+#include <iostream>
+#include <string>
+#include <stack>
+
+/**
+ * @brief Get and pop last two items on the stack. This is done because std::stack::pop does not return value that is removed
```

Comment:
also add information about it being commonly reused through the code?

Reply:
why?

Reply:
thats fine then